packages/allure-js-commons/src/facade.ts

```diff
@@ -76,6 +76,12 @@ export const attachment = (
   return callRuntimeMethod("attachment", name, content, opts);
 };
 
+export const attachTrace = (name: string, path: string) => {
```

Comment:
It shouldn't be a part of the facade due to the trace is a playwright's only abstraction